vm/src/import.rs

```diff
@@ -23,6 +25,15 @@ pub fn init_importlib(vm: &VirtualMachine, external: bool) -> PyResult {
         let install_external =
             vm.get_attribute(importlib.clone(), "_install_external_importers")?;
         vm.invoke(install_external, vec![])?;
+        // Set pyc magic number to commit hash. Should be changed when bytecode will be more stable.
+        let sys_modules = vm.get_attribute(vm.sys_module.clone(), "modules")?;
```

Comment:
Any reason not to use `import_frozen()` here?

Reply:
Nevermind, that would make a completely new module object. What about `vm.import`? That checks `sys.modules` for the module just like you are here, I believe.

Reply:
Done